src/Microsoft.Diagnostics.Runtime/Native/NativeRuntime.cs

```diff
@@ -46,12 +50,22 @@ protected override void InitApi()
 
                 _sos = (ISOSNative)dac;
             }
+            if (_sosNativeSerializedExceptionSupport == null)
+            {
+                var dac = _library.DacInterface;
```

Comment:
You can replace all of this with: `_sosNativeSerializedExceptionSupport = _library.DacInterface as ISOSNativeSerializedExceptionSupport;`.